docs/source/usage/logger.rst

```diff
@@ -17,101 +17,76 @@ the logging system for advanced use-cases.
+#
+#    For example, if you want to see only the debug messages generated by
+#    ``saga.engine`` and a specific middleware adaptor called ``xyz`` you would
+#    set the following environment variables::
+#
+#        SAGA_VERBOSE=DEBUG SAGA_LOG_FILTERS=saga.engine,saga.adaptor.xyz python mysagaprog.py
 
-   Controls the message sources displayed. SAGA uses a 
-   hierarchal structure for its log sources. Starting with the root logger
-   ``saga``, several sub loggers are defined for SAGA-internal logging events
-   (``saga.engine``) and individual middleware adaptors ``saga.adaptor.name``.
-   ``SAGA_LOG_FILTERS`` expects either a single source name or a 
-   comma-separated list of source names. Non-existing source names are ignored.
 
-   For example, if you want to see only the debug messages generated by 
-   ``saga.engine`` and a specific middleware adaptor called ``xyz`` you would 
-   set the following environment variables::
+.. envvar:: RADICAL_SAGA_LOG_TGT
 
-       SAGA_VERBOSE=DEBUG SAGA_LOG_FILTERS=saga.engine,saga.adaptor.xyz python mysagaprog.py
-
-
-.. envvar:: SAGA_LOG_TARGETS
-
-   Controls where the log messages go. Multiple concurrent locations are 
-   supported. 
-   ``SAGA_LOG_TARGETS`` expects either a single location or a comma-separated 
+   Controls where the log messages go. Multiple concurrent locations are
+   supported.
+   ``SAGA_LOG_TARGETS`` expects either a single location or a comma-separated
    list of locations, where a location can either be a path/filename or
-   the ``STDOUT`` keyword (case insensitive) for logging to the console. 
+   the ``stdout``/``stderr`` keyword (case sensitive) for logging to the console.
 
    For example, if you want to see debug messages on the console but also
-   want to log them in a file for further analysis, you would set the the 
+   want to log them in a file for further analysis, you would set the the
    following environment variables::
 
-       SAGA_VERBOSE=DEBUG SAGA_LOG_TARGETS=STDOUT,/tmp/mysaga.log python mysagaprog.py
-
-
-.. _log_api:
-
-Application Level Logging
--------------------------
-
-The RADICAL-SAGA logging utilities are a thin wrapper around Python's logging
-facilities, integrated into the RADICAL-SAGA configuration facilities.
-To support the seamless integration of application level logging needs, the
-:func:`saga.utils.logger.getLogger` allows to produce additional logger
-facilities, which are again native Python :class:`logging.Logger` instances, but
-preconfigured according to the RADICAL-SAGA logging configuration.
-Those instances can then be further customized as needed::
-
-   from saga.utils.logger import getLogger, INFO
-
-   app_logger       = getLogger ('application.test')
-   app_logger.level = INFO
-
-   app_logger.info ('application level log message on INFO level')
-
-
-.. automodule:: saga.utils.logger
-   :members:
+       SAGA_VERBOSE=DEBUG SAGA_LOG_TARGETS=stdout,./rs.log python mysagaprog.py
```

Comment:
`RADICAL_SAGA_LOG_TGT` or `SAGA_LOG_TARGETS`

Reply:
Thanks, fixed (and some other items in that file corrected)